src/services/TitleService.js

```diff
@@ -0,0 +1,13 @@
+import Service from '../abc/Service';
+export default class TitleService extends Service {
+	constructor(app, serviceName="TitleService"){
+		super(app, serviceName);
+		this.App = app;
+	}
+
+	setTitle = (subTitle) => {
+		let title = this.App.Config.get("title");
```

Comment:
@Pe5h4  I moved the title assignment from the constructor, because if I got it there, it returned undefined to me. But if I get it in setTitle, it returns title

Reply:
@Aringoaway Yes, that is correct. It should not be in the constructor. However, it should be in `initialize` method, because you dont need to request the title everytime again and again. Its totally fine to get it once during the initialization.

```
initialize() {
this.title = this.App.Config.get("title");
}
```